examples/scripts/3d_equation_cylindric.py

```diff
@@ -0,0 +1,212 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.interpolate import griddata
+
+import pybamm
+from pybamm import (
+    BaseModel,
+    Discretisation,
+    Scalar,
+    ScikitFiniteElement3D,
+    SpatialVariable,
+)
+
+R = 0.5
+H = 1.0
+alpha = 0.1
+
+model = BaseModel()
+
+x = SpatialVariable("x", ["current collector"], coord_sys="cartesian", direction="x")
```

Comment:
Maybe I'm misunderstanding, but I'm confused as to why we aren't using cylindrical variables here? I was expecting to do something like solve the heat equation in cylindrical coordinates in 3D.